apps/mark-scan/frontend/src/pages/ballot_invalidated_page.tsx

```diff
@@ -0,0 +1,44 @@
+/* istanbul ignore file - placeholder component that will change */
+import { useContext } from 'react';
+
+import { Screen, H1, Main, Button, Text } from '@votingworks/ui';
+
+import { confirmInvalidateBallot } from '../api';
+
+import { BallotContext } from '../contexts/ballot_context';
+import { ButtonFooter } from '../components/button_footer';
+
+export function BallotInvalidatedPage(): JSX.Element | null {
+  const { resetBallot, endVoterSession } = useContext(BallotContext);
+
+  const confirmInvalidateBallotMutation = confirmInvalidateBallot.useMutation();
+
+  function onPressContinue() {
+    confirmInvalidateBallotMutation.mutate(undefined, {
+      async onSuccess() {
+        resetBallot();
+        await endVoterSession();
+      },
+    });
+  }
+
+  return (
+    <Screen>
+      <Main padded centerChild>
+        <Text center>
+          <H1>
+            <span aria-label="Ballot Invalidated.">Ballot Invalidated</span>
```

Comment:
Since most people generally don't read the fine print, I might consider changing this headline to something like "Ask a Poll Worker for Help" so that it's clear what action is important.